app/models/location.rb

```diff
@@ -40,14 +41,24 @@ def set_phone
     self.phone = Phonifyer.phonify(self.phone) if self.phone
   end
 
+  def set_organization_coordinates
```

Comment:
Things I'm worried about - Location updating, organization doesn't update coordinates
- Organization has multiple locations, it's unclear which one sets the coordinates for the organization.

Does putting this method on `organization` instead of location make more sense? If we're already bumping the `updated_at`, seems like this would fit well in the `set_calculated_attributes` method on Organization. And also would make sense to utilize the `search_location` method.